core/src/main/scala/kafka/controller/TopicDeletionManager.scala

```diff
@@ -84,6 +83,12 @@ class TopicDeletionManager(controller: KafkaController) extends Logging {
     }
   }
 
+  def start(): Unit = {
```

Comment:
Having init(), start(), shutdown() together can be a bit confusing. How about rename them to init(), tryTopicDeletion() and reset()?